### ISSUE TITLE
Update the docs

### ISSUE THREAD

**Issue by felipenoris** [felipenoris/hyper-reverse-proxy#36]
* [ ] rewrite docs main page with new examples
* [ ] static client
* [ ] ws support
* [ ] notes on security
* [ ] sync README.md with main page docs

**Comment by somehowchris** [comment]
Do yu mean the docs inside `lib.rs`?

**Comment by felipenoris** [comment]
@somehowchris , yes, the doc strings that generate https://docs.rs/hyper-reverse-proxy/latest/hyper_reverse_proxy/ and README.md which is a summary of the docs.

**Comment by somehowchris** [comment]
So #45 would include your points 1, 2, 4 & 5 inside the `README.md` and inside the mod doc, what kind of doc would you suggest for your point `ws support`. Just mentioning it?

**Comment by somehowchris** [comment]
@felipenoris So what's needed for a next release?

**Comment by somehowchris** [comment]
Hey @felipenoris,

It just makes me sad seeing such a lovely crate being unmaintained, people are loving it and opening up PRs to suggest awesome things. Do you plan to maintain it? If you need help, would love to chime in
